Survey/Configurations/Environment/Environment.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Environment.swift
+//  Survey
+//
+//  Created by Khanh on 01/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+
+enum Environment {
+
+    static func based<T>(staging: T, production: T) -> T {
+        #if DEBUG
```

Comment:
`DEBUG` is normally used for running from Xcode with debugging vs running with an `ipa` (Firebase Dist/Testflight). I suggest you add the flag `PROD` to use for this block.